src/js/index.js

```diff
@@ -1,2 +1,69 @@
 import 'bootstrap/dist/css/bootstrap';
 import $ from 'jquery';
+
+import requestAdaptor from './request.js';
+
+const request = new requestAdaptor();
+
+const main = function() {
+  let data = {};
+  if (arguments[0]) data = arguments[0];
+  const showChartName = data => {
+    let name = 'Bitcoin';
+    if (data.chartName) name = data.chartName;
+    $('#chartname').html(name);
+  };
+
+  const showDate = data => {
+    let date = '';
+    if (data.time && data.time.updated) date = data.time.updated;
+    $('#date-banner-data').html(date);
+  };
+
+  const showDisclaimer = data => {
+    let disclmr = '';
+    if (data.disclaimer) disclmr = data.disclaimer;
+    $('#disclmr').html(disclmr);
+  };
+  const showGrid = data => {
+    let prices = [];
+    if (data.bpi) {
+      let { bpi } = data;
+      for (let index in bpi) {
+        prices.push(bpi[index]);
+      }
+      prices.map((value, index) => {
+        let row = index % 2 === 0 ? $('<tr/>') : $("<tr class= 'dark-row'/>");
```

Comment:
This can be achieve by css selectors `:odd` and `:even`

Even better with bootstrap's `table-striped` CSS class.